.github/workflows/cygwin-windows.yml

```diff
@@ -0,0 +1,79 @@
+# Based on https://github.com/mit-plv/fiat-crypto/blob/master/.github/workflows/coq-windows.yml
+name: CI (Cygwin/Windows)
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    env:
+      CFLAGS: "--coverage -O2 -g"
+      CXXFLAGS: "--coverage -O2 -g"
+      LDFLAGS: "--coverage"
+      # default config flags: enable debug asserts
+      CONFIGFLAGS: "--enable-debug"
+      COVERALLS_PARALLEL: true
+      TEST_SUITES: testinstall
+
+    steps:
+    # This sets git to use Linux line endings. While GAP should allow
+    # Windows line endings in GAP files, we currently can't build GAP if the
+    # whole source has Windows line endings
+    - name: Set git to use LF
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+
+    - uses: actions/checkout@v2
+    - name: Download cygwin
```

Comment:
This step feels like something that could/should be in a Github Action of its own, perhaps under the gap-actions org. @ssiccha knows how to do that, in case you are interested, Chris. That would then make it easier for packages to use this, too.